test/rubygems/test_gem_installer.rb

```diff
@@ -1435,7 +1430,6 @@ def test_find_lib_file_after_install
 
     write_file File.join(@tempdir, "a.c") do |io|
       io.write <<-C
-        #include <ruby.h>
```

Comment:
This isn't needed and was causing problems without ruby installed, since it wasn't able to find the `ruby/config.h` header, since it wasn't finding it where it gets installed.  So I removed it to avoid breaking this test.

Reply:
Oh, looks like the MinGW CI failure is related to this, where it defaults to `--enable-shared` so can't find the library for linking (it does do `make install` but with `DESTDIR=..`, so it still doesn't have the library in the installation prefix).